From c5fde5a3c50897baecea97eb04f4dee62d3de814 Mon Sep 17 00:00:00 2001
From: Jerry Cao <jerry.cao@amlogic.com>
Date: Thu, 1 May 2014 12:28:40 -0700
Subject: [PATCH 3983/5965] [meson8] Update meson8 TrustZone kernel defconfig

    Sync up meson8_tee_defconfig (TrustZone enabled) with meson8_defconfig
---
 arch/arm/configs/meson8_tee_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/meson8_tee_defconfig b/arch/arm/configs/meson8_tee_defconfig
index 5ab9bb34d1e9..3d4c433b817a 100644
--- a/arch/arm/configs/meson8_tee_defconfig
+++ b/arch/arm/configs/meson8_tee_defconfig
@@ -41,6 +41,7 @@ CONFIG_UNIX=y
 CONFIG_UNIX_DIAG=y
 CONFIG_NET_KEY=y
 CONFIG_INET=y
+CONFIG_IP_MULTICAST=y
 CONFIG_IP_ADVANCED_ROUTER=y
 CONFIG_IP_MULTIPLE_TABLES=y
 CONFIG_INET_ESP=y
-- 
2.19.0

